Stripe/PublicHeaders/STPAddress.h

```diff
@@ -167,6 +167,24 @@ typedef NS_ENUM(NSUInteger, STPBillingAddressFields) {
 - (BOOL)containsRequiredFields:(STPBillingAddressFields)requiredFields;
 
 /**
+ Checks if this STPAddress has any content (possibly invalid) in any of the
+ desired billing address fields.
+
+ Where `containsRequiredFields:` validates that this STPAddress contains valid data in
+ all of the required fields, this method checks for the existence of *any* data.
+
+ For example, if `desiredFields` is `STPBillingAddressFieldsZip`, this will check
+ if the postalCode is empty.
+
+ Note: When `desiredFields == STPBillingAddressFieldsNone`, this method always returns
+ NO.
+
+ @parameter desiredFields The billing address information the caller is interested in.
+ @return YES if there is any data in this STPAddress that's relevant for those fields.
+ */
+- (BOOL)containsContentForFields:(STPBillingAddressFields)desiredFields;
```

Comment:
I think this should be `billingAddressFields`. It doesn't match the other one (which was written before billing address existed) but I think new methods going forward should make clear whether they are doing billing or shipping fields and at some point we can deprecate/rename the old ones. This is what we did for the newer Apple Pay billing address related method further down.